doc/tutorials/acrn-sanity-test.rst

```diff
@@ -0,0 +1,227 @@
+
+     $ cd /home/<your user name>/uos/
+
+     $ curl https://download.clearlinux.org/releases/xxxxx/clear/clear-xxxxx-kvm.img.xz -o uos.img.xz
+
+     $ unxz uos.img.xz
+
+     $ sudo sed -i "s|/home/clear|$HOME|g" /usr/share/acrn/samples/nuc/launch_uos.sh
+
+     $ sudo /usr/share/acrn/samples/nuc/launch_uos.sh
+
+* Expected Result
+
+User VM boot up normally
+
+ST_HV_Memory_LaaG_Default_Partition_Check
+*****************************************
+
+* Description
+
+This test case focus on checking if the pass through memory to User VM and
+check if the memory size is in a reasonable scope as ``launch_uos.sh`` defined.
+Since the User VM will take a part of memory, the available memory size will
+not be exact as what pass-through.
+
+* Prerequisites
+
+Service VM and User VM boot up normally
+
+* Test Steps
+
+1. Boot to Service VM
+#. Boot User VM with default memory setting in ``launch_uos.sh``
+#. Check ``cat /proc/meminfo`` and see if the available memory is close the size passed through to User VM
+
+Expected Result
+
+1. Boot User VM successfully
+#. memory is close the size passed through to User VM
+
+IO
+**
+
+ST_ACRNIO_LaaG_Virtio_net
+*************************
+
+* Description
+
+This test case focus on checking if Service VM and User VM can be allocated different IP,
+and ping is ok by sharing same physical network card.
+
+* Prerequisites
+
+VirtIO and bridge/tap is created normally in Service VM
+
+* Test Steps
+
+1. Boot the Device Under Test (DUT), and enter Service VM from the ACRN hypervisor console (serial port).
+
+   .. code-block:: none
+
+      ACRN:\>vm_console
+
+#. Check IP and bridge/tap after log in Service VM.
+
+   .. code-block:: none
+     
+      $ ip a
+      $ brctl show
+      bridge name     bridge id               STP enabled     interfaces
+      acrn-br0         8000.1ad4ac453b20       no              acrn_tap0
+                                                               ens4
+
+#. Boot User VM using the ``launch_uos.sh`` and check IP address.
+
+   .. code-block:: none
+   
+      $ sudo /usr/share/acrn/samples/nuc/launch_uos.sh
+
+      $ ip a
+
+* Expected Result
+
+IP address should be allocated for both Service VM  and User VM, and ping is ok
+
+
+ST_ACRNIO_SOS_Storage
+*********************
+
+* Description
+
+This test case focus on checking if storage device works correctly in Service VM.
+
+* Prerequisites
+
+storage device is installed
+
+* Test Steps
+
+1. Boot into Service VM
+
+   .. code-block:: none
+  
+   $ fdisk -l 
+
+#. Find storage device and mount it to read/write
+
+   * SATA -> /dev/sda
+   * NVMe -> /dev/nvme0n1
+   * eMMC -> /dev/mmcblk0
+
+* Expected Result
+
+storage device be used normally in Service VM
+
+Graphics Virtualization (GVT)
+*****************************
+
+ST_ACRNGT_SOS_Rendering_Weston_APP
+**********************************
+
+* AcrnGT provide virtual display on Service VM: run weston weston-simple-egl in Service VM,
```

Comment:
This one and the next one could do with a bit more explanations. Both start off with the exact same steps and the only difference is that one uses `weston-simple-egl` and the other `weston-simple-shm`. What are these testing really?

Reply:
Will display different picture with "weston-simple-egl" and "weston-simple-shm"

Reply:
I do not quite understand your response here? Are you saying that both will display different graphics artefacts on the screen?

I do not know the details but I believe both use different mechanisms to display things on the display, this is what I'm interested in, i.e. having a short description of that.